manual_scripts/active_directory/check_azure_users.py

```diff
@@ -0,0 +1,72 @@
+# Python script to check whether a list of users exists in DLUHC Azure AD.
+# Used during the SSO switchover to make sure we are not going to lock users out.
+# Needs a valid bearer token for Microsoft Graph
+
+# Expects a list of user emails in prod_dluhc_users.txt, can get this from the AD Management Server (Powershell)
+# $date = (get-date).AddDays(-90)
+# $fdate = $date.tofiletime()
+# Get-ADUser -Filter {(LastLogonTimeStamp -gt $fdate -AND enabled -eq $true -AND mail -like "*@levellingup.gov.uk")} -Properties mail | Select-Object -Property mail
```

Comment:
Because you separately checked for @communities.gov.uk users and updated them?

Reply:
Yeah I've checked on Delta manually, there's two accounts that are enabled, I've already talked to Sudheer, one other person I need to message, but nothing will change for them anyway.